src/main/features/dashboard/views/macro/claimStatus/statesPaidRejected.njk

```diff
@@ -40,19 +54,34 @@
   {% else %}
     {% set amount = claim.response.paymentDeclaration.paidAmount  %}
   {% endif %}
-  <h2 class="heading-small">{{ t('{{ claimantName }} rejected your admission of {{ amount }}', {
-      claimantName: claim.claimData.claimant.name,
-      amount: amount | numeral
-    }) }}</h2>
+  {% set mediation = claim.claimantResponse.freeMediation and claim.claimantResponse.freeMediation === YesNoOption.YES.option %}
+
+  {% if mediation %}
+    <h2 class="heading-small">{{ t('We’ll contact you with a mediation appointment') }}</h2>
+  {% else %}
+    <h2 class="heading-small">{{ t('The claimant has rejected your admission of {{ amount }}', { amount: amount | numeral}) }}</h2>
```

Comment:
I think we need to display the claimant's name here according to the prototype

Reply:
As discussed, this matches the January prototype I was instructed to follow. UX should be updating the FTBB prototype accordingly.